shared-libs/outbound/.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "env": {
+    "node": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+  }
+}
```

Comment:
So that in the test I could write:

```
const service = outbound({... console, isDebugEnabled: () => false});
```

As a quick and clean way of providing a mock logger. This is supported by Node 8, but you're right, I'm not sure if the entirety of ecma2018 is. I've moved the eslint file to the test dir, so now it's scoped to dev environments only.